crates/uv-python/src/interpreter.rs

```diff
@@ -713,6 +714,42 @@ pub enum InterpreterInfoError {
     },
 }
 
+/// Environment variables that can change the values of [`InterpreterInfo`].
+#[derive(Debug, Deserialize, Serialize, Clone, PartialEq, Eq)]
+struct PythonEnvVars {
+    /// `PYTHONHOME` overrides `sys.prefix`.
+    pythonhome: Option<OsString>,
+    /// `PYTHONPATH` adds to `sys.path`.
+    pythonpath: Option<OsString>,
+    /// `PYTHONSAFEPATH` influences `sys.path`.
+    pythonsafepath: Option<OsString>,
+    /// `PYTHONPLATLIBDIR` influences `sys.path`.
+    pythonplatlibdir: Option<OsString>,
+    /// `PYTHONNOUSERSITE` influences `sys.path`.
+    pythonnousersite: Option<OsString>,
+    /// `PYTHONUSERBASE` influences `sys.path`.
+    pythonuserbase: Option<OsString>,
+    /// `APPDATA` influences `sys.path` through the user site packages (windows).
+    appdata: Option<OsString>,
+    /// `HOME` influences `sys.path` through the user site packages (unix).
+    home: Option<OsString>,
+}
+
+impl PythonEnvVars {
+    fn from_env() -> Self {
+        Self {
+            pythonhome: env::var_os("PYTHONHOME"),
+            pythonpath: env::var_os("PYTHONPATH"),
+            pythonsafepath: env::var_os("PYTHONSAFEPATH"),
+            pythonplatlibdir: env::var_os("PYTHONPLATLIBDIR"),
+            pythonnousersite: env::var_os("PYTHONNOUSERSITE"),
+            pythonuserbase: env::var_os("PYTHONUSERBASE"),
+            appdata: env::var_os("APPDATA"),
+            home: env::var_os("HOME"),
```

Comment:
Technically, we should add all these to the static `EnvVars` declarations so we can document what we read these for. Could you do that please?

Reply:
oops, good catch!